### PR TITLE
[gha] testing

### PR DESCRIPTION
**Pull request by rexhoffman**
## Motivation

Check tests on gha in master w/o changes.

### Have you read the [Contributing Guidelines on pull requests]

Y

## Test Plan

CI

## Related PRs

None

### CODE REVIEW

**Comment by rexhoffman** [comment]
So looks like removing the slash I added to work around another problem now works...   I was very bleary eyed when I updated that to use that slash, but hard to argue with the passing tests (as far as we trust the test).

File: .github/workflows/ci-test.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ env:
   max_threads: 16
   pre_command: cd /opt/git/diem/
 
+
```
Comment (bmwill) [inline .github/workflows/ci-test.yml:17]: remove this extra line before merging

**Review by bmwill — APPROVED** [review]

**Pull request closed without merging**